### PR TITLE
Add FAQ point regarding denomination

### PR DESCRIPTION
**Pull request by cmichi**

### CODE REVIEW

**Review by Robbepop — APPROVED** [review]
LGTM

Good addition!

File: docs/faq/faq.md [hunk 1]
```diff
@@ -217,3 +217,24 @@ can also be used to operate on the contract's storage, however, they will load t
...
+
+Floats are cool for all kinds of reasons, but they also have one important
+drawback. Floating point arithmetic is non-deterministic which means that
+different processors compute (slightly) different results for the same
+operation. Although there is an IEEE spec, non-determinism can come from specific
+libraries used, or even hardware. In order for the nodes in a blockchain network
+to reach agreement on the state of the chain, all operations must be completely
+deterministic. Hence we don't allow floating point data types in ink!.
+
+Subsequently it's not possible to return a decimal number from an ink! message.
```
Comment (ascjones) [inline docs/faq/faq.md:232]:
```suggestion
Consequently it's not possible to return a decimal number from an ink! message.
```

File: docs/faq/faq.md [hunk 2]
```diff
@@ -217,3 +217,24 @@ can also be used to operate on the contract's storage, however, they will load t
...
+What you should do instead is to have your user interface denominate the returned
+number to decimals.
+
+Note, that it's typical for blockchains to have the number of available tokens
+defined as a non-floating number and determine the decimal places in the user
+interface. For example, 1 Bitcoin is equivalent to the smallest unit of 100.000.000
```
Comment (ascjones) [inline docs/faq/faq.md:238]:
```suggestion
interface. For example, 1 Bitcoin is equivalent to the smallest unit of 100,000,000
```

File: docs/faq/faq.md [hunk 3]
```diff
@@ -217,3 +217,24 @@ can also be used to operate on the contract's storage, however, they will load t
...
+libraries used, or even hardware. In order for the nodes in a blockchain network
+to reach agreement on the state of the chain, all operations must be completely
+deterministic. Hence we don't allow floating point data types in ink!.
+
+Subsequently it's not possible to return a decimal number from an ink! message.
+What you should do instead is to have your user interface denominate the returned
+number to decimals.
+
+Note, that it's typical for blockchains to have the number of available tokens
+defined as a non-floating number and determine the decimal places in the user
```
Comment (ascjones) [inline docs/faq/faq.md:237]:
```suggestion
defined as a non-floating number and determine the denomination in the user
```

**Review by athei — APPROVED** [review]

**Pull request merged**